.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "node": true,
+        "commonjs": true,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": "latest"
+    },
+    "rules": {
```

Comment:
We should probably just remove this field if it's empty

Reply:
This was auto generated by  `npm init @eslint/config` so I don't really feel like like editing it. Gonna merge now to speed things up. Let me know if you have a strong opinion.